.github/workflows/conda_canary.yml

```
@@ -27,7 +27,8 @@ jobs:
   conda_nightly:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
+        ref: '1.x'  # Conda latest is on 1.x version
```

Shouldn't it be this

```suggestion
        with:
          ref: '1.x'  # Conda latest is on 1.x version
```